### PR TITLE
Fixed ipad view in skills category

### LINKED ISSUE #2655: Fix ipad view in skills category
**Actual Behaviour**

Ipad view is not proper.

**Expected Behaviour**

It should be proper.

**Screenshots of the issue**

Capture

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2655

Changes: Fixed ipad and ipad pro view in skills category

Demo Link: https://pr-2660-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/cms/SkillCardList/SkillCardList.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const ImageContainer = styled.div`
 
 const DetailsContainer = styled.div`
   width: auto;
-  min-width: 45rem;
+  // min-width: 45rem;
```
Comment (harsh-jindal) [inline src/components/cms/SkillCardList/SkillCardList.js]: @samagragupta Why is this commented ?

**Comment by harsh-jindal** [comment]
Increase space between two test skill examples boxes.
Screenshot 2019-07-24 at 12 34 26 AM

**Comment by samagragupta** [comment]
@harsh-jindal Fixed that.

**Comment by fragm3** [comment]
> Increase space between two test skill examples boxes.
> Screenshot 2019-07-24 at 12 34 26 AM

Not fixed

**Comment by samagragupta** [comment]
@fragm3 Fixed

File: src/components/cms/BrowseSkill/BrowseSkill.js [hunk 2]
```diff
@@ -879,7 +879,7 @@ class BrowseSkill extends React.Component {
                             <Select
                               value={entriesPerPage}
                               onChange={this.handleEntriesPerPageChange}
-                              style={{ width: '5.1rem' }}
+                              style={{ width: '5.1rem', marginTop: '22px' }}
```
Comment (akshatnitd) [inline src/components/cms/BrowseSkill/BrowseSkill.js]: Any reason to use `px` here?

Comment (samagragupta) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: @akshatnitd Changed it to rem.

Comment (samagragupta) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: @akshatnitd Changed it to rem.

**Pull request merged**